Controller/src/controller/rpc.go

```diff
@@ -70,13 +70,13 @@ func makeCert() error {
 	if err != nil {
 		return err
 	}
-	config.Metadata.Cert = cert
+	*config.Metadata.Cert = string(cert)
```

Comment:
config.GetMetadta().Cert = proto.String(string(cert)) ?

Reply:
addressed in 024773b